webdriver-spec.html

```diff
@@ -6546,185 +6556,327 @@
       </ol>
 
     </section>
-    <!-- <REMOVE> -->
 
-  <section><!-- Pointer Actions -->
-    <h2><dfn data-lt=pointer>Pointer Actions</dfn></h2>
-    <p>The <dfn>pointerUp</dfn> command:
-      <ol>
-        <li><p>If the <a>current top-level browsing context</a> is <a>no longer open</a>
-          , return <a>error</a> with <a>error code</a> <a>no such window</a>.
+    <section> <!-- Pointer Actions -->
+      <h2>Pointer Actions</h2>
 
-        <li><p><a>Handle any user prompts</a> and return its value if it is an
-          <a>error</a>.
+      <p>When required to <dfn data-lt="key-state">get a global key
+        state</dfn> with argument <var>key</var>, a remote end must
+        run the following steps:
+        <ol>
+          <li>Let <var>input state</var> be the <a>current
+              session</a>'s <a>input state</a> object.
 
-        <li><p>If <var>element id</var> is <a>undefined</a> jump to step 9.
+          <li>For each <var>device input state</var> that is a value
+            of the <var>input state</var> map:
 
-        <li><p>Let <var>element result</var> be the result of
-         <a>getting a known element</a> by
-         <a>UUID</a> reference <var>element id</var>.
+            <ol>
+              <li>If <var>device input state</var> is a <a>key input
+              state</a> object, and its <code>pressed</code> property
+                contains <var>key</var> return true.
+            </ol>
```

Comment:
A matter of taste, but using an iterator with a child list could be replaced by a paragraph saying something along the lines of:

> If any _device input state_ in the _current session_’s _input state_ is a _key input state_ object and its `pressed` property contains _var_, return true.

Anyway, this is not a blocker.